contrib/builder/deb/amd64/generate.sh

```diff
@@ -76,7 +76,6 @@ for version in "${versions[@]}"; do
 		libdevmapper-dev # for "libdevmapper.h"
 		libltdl-dev # for pkcs11 "ltdl.h"
 		libseccomp-dev  # for "seccomp.h" & "libseccomp.so"
-		libsqlite3-dev # for "sqlite3.h"
```

Comment:
ah, I thought I had but you are right it didn't actually generate them as I was on amd64...